recipes-kernel/kernel-tests/kernel-tests.bb

```diff
@@ -4,84 +4,86 @@ SECTION = "tests"
 }
 
-do_install_ptest_append() {
-    cp ${WORKDIR}/run-ptest ${D}${PTEST_PATH}
-    cp ${WORKDIR}/ptest-format.sh ${D}${PTEST_PATH}
-    cp ${WORKDIR}/security-mitigations.txt ${D}${PTEST_PATH}
-    cp ${WORKDIR}/test_kernel_mcopy.sh ${D}${PTEST_PATH}
-    cp ${WORKDIR}/test_kernel_mcopy_functionality ${D}${PTEST_PATH}
-    cp ${WORKDIR}/test_kernel_mcopy_freed_memory ${D}${PTEST_PATH}
-    cp ${WORKDIR}/test_kernel_cache_info.sh ${D}${PTEST_PATH}
-    cp ${WORKDIR}/test_kernel_cap_support.sh ${D}${PTEST_PATH}
-    cp ${WORKDIR}/test_kernel_modules.sh ${D}${PTEST_PATH}
-    cp ${WORKDIR}/test_kernel_security.sh ${D}${PTEST_PATH}
-    cp ${WORKDIR}/test_exe_cap_support ${D}${PTEST_PATH}
-    cp ${WORKDIR}/cap_support_exe_to_test ${D}${PTEST_PATH}
-    cp ${WORKDIR}/test_proc_cap_support ${D}${PTEST_PATH}
-    cp ${WORKDIR}/test_kernel_ll_route.sh ${D}${PTEST_PATH}
-    cp ${WORKDIR}/test_kernel_hrtimers.sh ${D}${PTEST_PATH}
-    cp ${WORKDIR}/test_kernel_hrtimers ${D}${PTEST_PATH}
-    cp ${WORKDIR}/test_user_stack_size.sh ${D}${PTEST_PATH}
-    cp ${WORKDIR}/test_pthread_stack_size ${D}${PTEST_PATH}
-    cp ${WORKDIR}/test_kernel_swap_disabled.sh ${D}${PTEST_PATH}
-    cp ${WORKDIR}/test_i915_firmware.sh ${D}${PTEST_PATH}
+do_install_ptest:append() {
+	cp ${WORKDIR}/run-ptest ${D}${PTEST_PATH}
+	cp ${WORKDIR}/ptest-format.sh ${D}${PTEST_PATH}
+	cp ${WORKDIR}/security-mitigations.txt ${D}${PTEST_PATH}
+	cp ${WORKDIR}/test_kernel_mcopy.sh ${D}${PTEST_PATH}
+	cp ${WORKDIR}/test_kernel_mcopy_functionality ${D}${PTEST_PATH}
+	cp ${WORKDIR}/test_kernel_mcopy_freed_memory ${D}${PTEST_PATH}
+	cp ${WORKDIR}/test_kernel_cache_info.sh ${D}${PTEST_PATH}
+	cp ${WORKDIR}/test_kernel_cap_support.sh ${D}${PTEST_PATH}
+	cp ${WORKDIR}/test_kernel_modules.sh ${D}${PTEST_PATH}
+	cp ${WORKDIR}/test_kernel_security.sh ${D}${PTEST_PATH}
+	cp ${WORKDIR}/test_exe_cap_support ${D}${PTEST_PATH}
+	cp ${WORKDIR}/cap_support_exe_to_test ${D}${PTEST_PATH}
+	cp ${WORKDIR}/test_proc_cap_support ${D}${PTEST_PATH}
+	cp ${WORKDIR}/test_kernel_ll_route.sh ${D}${PTEST_PATH}
+	cp ${WORKDIR}/test_kernel_hrtimers.sh ${D}${PTEST_PATH}
+	cp ${WORKDIR}/test_kernel_hrtimers ${D}${PTEST_PATH}
+	cp ${WORKDIR}/test_user_stack_size.sh ${D}${PTEST_PATH}
+	cp ${WORKDIR}/test_pthread_stack_size ${D}${PTEST_PATH}
+	cp ${WORKDIR}/test_kernel_swap_disabled.sh ${D}${PTEST_PATH}
+	cp ${WORKDIR}/test_i915_firmware.sh ${D}${PTEST_PATH}
 }
 
-do_install_ptest_append_x64() {
-    cp ${WORKDIR}/required_kernel_modules.x64 ${D}${PTEST_PATH}/required_kernel_modules
+do_install_ptest:append_x64() {
+	cp ${WORKDIR}/required_kernel_modules.x64 ${D}${PTEST_PATH}/required_kernel_modules
 }
 
-do_install_ptest_append_xilinx-zynqhf() {
-    cp ${WORKDIR}/required_kernel_modules.arm ${D}${PTEST_PATH}/required_kernel_modules
+do_install_ptest:append_xilinx-zynqhf() {
+	cp ${WORKDIR}/required_kernel_modules.arm ${D}${PTEST_PATH}/required_kernel_modules
```

Comment:
Could you write a work item to (1) get rid of the ARM bits here and (2) fold the x64 stuff into the default `do_install_ptest`?

Reply:
azdo: [2116507 ](https://dev.azure.com/ni/DevCentral/_workitems/edit/2116507)